app/views/tmdb/tv_series.html.erb

```diff
@@ -1,40 +1,49 @@
 <article id="profile">
```

Comment:
I recommend turning off whitespaces in the github diff view to see this file best.